### PR TITLE
Docs: Update enter/shift+enter details for prometheus query field

### PR DESCRIPTION
**Pull request by gelicia**
A user on twitter brought up inconsistencies between our UI and the PrometheusIO UI. We will want to investigate that further, but he also brought up our docs say we are consistent when we actually are not. This updates the docs to be accurate with current behavior.

Source: https://twitter.com/kfekete/status/1550094074618826752

### CODE REVIEW

**Comment by gelicia** [comment]
Added it to the docs board.

File: docs/sources/explore/query-management.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ On the left side of the query field, click **Metrics** to open the Metric Explor
 
 ### Query field
 
-The Query field supports autocomplete for metric names, function and works mostly the same way as the standard Prometheus query editor. Press the enter key to execute a query.
+The Query field supports autocomplete for metric names, function and works mostly the same way as the standard Prometheus query editor. Press the enter key to create a new line, and shift+enter to execute a query.
```
Comment (knylander-grafana) [inline docs/sources/explore/query-management.md:98]: Does the autocomplete work for metric names and functions? Or does the sentence mean that functions and usage are mostly the same as the Prometheus query editor. If that's the case, here's a suggestion. What do you think? 
```suggestion
The Query field supports autocomplete for metric names. The editor has functions comparable to the standard Prometheus query editor. Press the Enter key to create a new line and Shift+Enter to execute a query.
```

**Review by knylander-grafana — APPROVED** [review]
Approving with a minor suggestion. Thank you for handling input from customers.

File: docs/sources/explore/query-management.md (hunk 1 above)
Comment (gelicia) [reply docs/sources/explore/query-management.md:98]: @knylander-grafana I think metric names and functions are equivalent in this sentence. What do you think about "The Query field supports autocomplete for metric names and functions, comparable to the standard Prometheus query editor."

Comment (knylander-grafana) [reply docs/sources/explore/query-management.md:98]: That sounds great!

**Pull request merged**